cmd/internal/ec2/add.go

```diff
@@ -63,6 +72,54 @@ func setupAgentForServer(collectionId string) error {
 	return nil
 }
 
+func askToReconfigure() error {
+	printer.Infof("postman-lc-agent is already present as a systemd service\n")
+	printer.Infof("Helpful commands \n Check status: systemctl status postman-lc-agent \n Disable agent: systemctl disable --now postman-lc-agent \n Check Logs: journalctl -fu postman-lc-agent\n Check env file: cat %s \n Check systemd service file: cat %s \n", envFilePath, serviceFilePath)
+	err := survey.AskOne(
+		&survey.Input{
+			Message: "Over-write old apiKey and collectionId with current values ? (y/n)",
+			Help:    "Enter y(default) or n.",
```

Comment:
The help should give a longer description of what we're asking them to do.  Something like:

"Should we rewrite the systemd configuration file with the current values of the API key and collection ID?"

I don't know if we can put the specific values here -- parsing the old file to show the change is probably too much work.

I haven't got there yet, the sentence above may not be good because if we're really overwriting then it's not just an update, so the 'help' message or event should make that clear that any customizations made to the file will be discarded.